test/test_immersed_advection.jl

```diff
@@ -74,20 +74,20 @@ function run_momentum_interpolation_test(u, v, ibg, scheme)
 
     for i in 7:19, j in 7:19
         if typeof(scheme) <: Centered
-            @test CUDA.@allowscalar  _symmetric_interpolate_xᶜᵃᵃ(i+1, j, 1, ibg, scheme, u, 1) ≈ 1.0
-            @test CUDA.@allowscalar  _symmetric_interpolate_xᶜᵃᵃ(i+1, j, 1, ibg, scheme, v, 1) ≈ 1.0
-            @test CUDA.@allowscalar  _symmetric_interpolate_yᵃᶜᵃ(i, j+1, 1, ibg, scheme, u, 1) ≈ 1.0
-            @test CUDA.@allowscalar  _symmetric_interpolate_yᵃᶜᵃ(i, j+1, 1, ibg, scheme, v, 1) ≈ 1.0
```

Comment:
what was the `, 1` doing here in the first place?

Reply:
I think it was a bug but how the `_symmetric_interpolate_xᶜᵃᵃ` it was not impacting the solution

Reply:
okie